scripts/setup_fullnode.sh

```diff
@@ -116,20 +118,43 @@ echo "Update your externalHost,producerPrivKey to config.yaml"
 sed -i "/^network:/a\ \ $externalHost" $IOTEX_HOME/etc/config.yaml
 sed -i "/^chain:/a\ \ $producerPrivKey" $IOTEX_HOME/etc/config.yaml
 
-echo -e "docker run iotex: ${YELLOW} ${version} ${NC}"
-#Run the following command to start a node:
-docker run -d --restart on-failure --name iotex \
-        -p 4689:4689 \
-        -p 8080:8080 \
-        -v=$IOTEX_HOME/data:/var/data:rw \
-        -v=$IOTEX_HOME/log:/var/log:rw \
-        -v=$IOTEX_HOME/etc/config.yaml:/etc/iotex/config_override.yaml:ro \
-        -v=$IOTEX_HOME/etc/genesis.yaml:/etc/iotex/genesis.yaml:ro \
-        iotex/iotex-core:${version} \
-        iotex-server \
-        -config-path=/etc/iotex/config_override.yaml \
-        -genesis-path=/etc/iotex/genesis.yaml
-
-#check node running
-sleep 5
-docker ps | grep iotex-server
+if [ "${wantmonitor}"X = "Y"X -o "${wantmonitor}"X = "y"X -o \
+	 "${wantmonitor}"X = "yes"X -o "${wantmonitor}"X = "Yes"X ];then
+    echo "Download config files for monitor"
+    mkdir -p $IOTEX_HOME/monitor
+    IOTEX_MONITOR_HOME=$IOTEX_HOME/monitor
+    curl -Ss https://raw.githubusercontent.com/iotexproject/iotex-bootstrap/${version}/monitor/Dockerfile > $IOTEX_MONITOR_HOME/
```

Comment:
```
Download config files for monitor
./scripts/setup_fullnode.sh: line 126: /Users/yutong/iotex-var/monitor/: Is a directory
./scripts/setup_fullnode.sh: line 127: /Users/yutong/iotex-var/monitor/: Is a directory
./scripts/setup_fullnode.sh: line 128: /Users/yutong/iotex-var/monitor/: Is a directory
./scripts/setup_fullnode.sh: line 129: /Users/yutong/iotex-var/monitor/: Is a directory
./scripts/setup_fullnode.sh: line 130: /Users/yutong/iotex-var/monitor/: Is a directory
./scripts/setup_fullnode.sh: line 131: /Users/yutong/iotex-var/monitor/: Is a directory
./scripts/setup_fullnode.sh: line 132: /Users/yutong/iotex-var/: Is a directory
chmod: /Users/yutong/iotex-var/monitor/run.sh: No such file or directory
```
I think if you use `>` you need to provide a file instead of a directory.